DllImportGenerator/DllImportGenerator.UnitTests/TestUtils.cs

```diff
@@ -95,7 +119,7 @@ public static (ReferenceAssemblies, MetadataReference) GetReferenceAssemblies()
                     "net6.0",
                     new PackageIdentity(
                         "Microsoft.NETCore.App.Ref",
-                        "6.0.0-preview.6.21317.4"),
+                        "6.0.0-preview.7.21376.23"),
```

Comment:
📝 This version doesn't exist:
https://www.nuget.org/packages/Microsoft.NETCore.App.Ref/

Reply:
It exists on the net6 Azure DevOps feed, which I think is where my machine pulled it from...

I guess we'll wait to merge this until Preview 7 comes out and I can update to a release version of P7.

Reply:
https://github.com/dotnet/roslyn-sdk/pull/639 was supposed to enable modified configurations, but if the APIs you need are present in the public release I would recommend using that. There is no need to have the current project build target match the reference assemblies used in tests.